app/assets/stylesheets/topics.scss

```diff
@@ -1,4 +1,4 @@
-.topics {
+.topic-section-list {
```

Comment:
:+1:

How about renaming the file to topic-section-list.scss? I like it when the block name is reflected in the filename. We follow that pattern in service-manual-frontend. Not that important so do what feels right.